### DISCUSSION TITLE
Minimum Version TLS

### DISCUSSION THREAD

**Discussion by masoud-shahsavari** [FranzKafkaYu/x-ui#177] [Ideas]
please add "min Version TLS feature" in the ui panel for tls setting.
"minVersion": "1.3" , "1.2" , "1.1"

**Comment by FranzKafkaYu** [comment]
> please add "min Version TLS feature" in the ui panel for tls setting. "minVersion": "1.3" , "1.2" , "1.1"

it's easy but do you really need this?

**Reply by masoud-shahsavari** [reply]
> > please add "min Version TLS feature" in the ui panel for tls setting. "minVersion": "1.3" , "1.2" , "1.1"
> 
> it's easy but do you really need this?

In Iran, if TLS below 1.2 is used, the server IP and domain will be blocked immediately, because it is assumed that most browsers and website use TLS 1.2 and above.

This feature prevents connecting to lower version TLS and the admin can choose the TLS version for client connecting.

**Reply by FranzKafkaYu** [reply]
> > > please add "min Version TLS feature" in the ui panel for tls setting. "minVersion": "1.3" , "1.2" , "1.1"
> > 
> > 
> > it's easy but do you really need this?
> 
> In Iran, if TLS below 1.2 is used, the server IP and domain will be blocked immediately, because it is assumed that most browsers and website use TLS 1.2 and above.
> 
> This feature prevents connecting to lower version TLS and the admin can choose the TLS version for client connecting.

I will add this in next version.

**Reply by masoud-shahsavari** [reply]
Thank you.
